src/artemis/system_tests/test_fgs_plan.py

```diff
@@ -79,12 +42,24 @@ def RE():
 
 @pytest.fixture
 def fgs_composite():
-    fast_grid_scan_composite = FGSComposite(
-        insertion_prefix=SIM_INSERTION_PREFIX,
-        name="fgs",
-        prefix=SIM_BEAMLINE,
-    )
-    fast_grid_scan_composite.wait_for_connection()
+    with patch("dodal.i03.dcm"):
+        with patch("dodal.i03.oav"):
```

Comment:
Nit: Maybe add a comment here that we're patching it cos there's no simulator. We should remove the patch once we have merged https://github.com/dls-controls/tickit-devices/pull/27

Reply:
removed this since they are removed from fgs composite anyway